src/admin-portal/create-campaign/create-technology/TechnologyDeals.jsx

```diff
@@ -1,12 +1,11 @@
 import React, { useState } from 'react'
 import Container from "react-bootstrap/Container";
-import { Button, Col, Row, Modal } from "react-bootstrap";
+import { Button, Col, Row, Modal, Button as BTN } from "react-bootstrap";
```

Comment:
Also, Not being nit-picky here but if you must rename a react component. pls ensure to follow the React component name casing convention, which is "PasCal" casing.